docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/uninstall.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_label: "Uninstallation"
+title: "Uninstall Palette"
+description: "Learn how to uninstall a Palette installation from your cluster using Helm charts."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["self-hosted", "enterprise"]
+keywords: ["self-hosted", "enterprise"]
+---
+
+To uninstall Palette from your cluster, you need to uninstall Palette management plane and Cert Manager. Optionally, you
+may also have installed Image Swap and Reach system, which also need to be uninstalled. This uninstall process applies
+to both connected and airgap instances of self-hosted Palette.
+
+:::warning
+
+This installation process only applies to Palette instances installed using Helm charts. If you used the Palette CLI to
+install Palette, this process does not apply.
+
+:::
+
+## Prerequisite
+
+- An active self-hosted Palette instance.
+
+- Access to the Kubernetes cluster where Palette is deployed.
+
+- Permissions to perform delete actions on all relevant cluster resources.
+
+- Helm is installed and available on your host machine.
+
+- kubectl is installed and available on your host machine.
+
+## Uninstall Palette
+
+1. Ensure you are using the appropriate context where Palette is deployed.
+
+   ```shell
+   kubectl config current-context
+   ```
+
+2. Issue the following command to start uninstalling the Palette management plane. Similar to Reach, this will only
+   remove the resources managed by Helm and the remaining resources will require additional manual intervention.
+
+   ```shell
+   helm uninstall hubble
+   ```
+
+3. Issue the following command to remove the namespace and custom resource definitions related to Palette management
+   plane.
+
+   ```shell
+   kubectl delete ns hubble-system || kubectl delete crd spectroclusteractions.jet.cluster.spectrocloud.com
+   ```
+
+4. Issue the following command to start uninstalling Reach. This will remove all resources related to Reach that are
+   managed by Helm. However, some resources created by Helm hooks are not managed by helm and will require additional
+   manual intervention to remove.
+
+   ```shell
+   helm uninstall reach-system
+   ```
+
+5. (Optional) If you installed Reach, issue the following command to start uninstalling Reach. This will remove all
+   resources related to Reach that are managed by Helm. However, some resources created by Helm hooks are not managed by
+   Helm and will require additional manual intervention to remove.
+
+   ```shell
+   helm uninstall reach-system
+   ```
+
```

Comment:
Did you mean to remove No.4?

Reply:
Yes, I accidentally copied over it. Thanks for spotting